sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs

```diff
@@ -4,8 +4,14 @@
 
 namespace Lusid.Sdk.Utilities
 {
+    /// <summary>
+    /// Builder class to build instances of ILusidApiFactory
+    /// </summary>
     public class LusidApiFactoryBuilder
     {
+        /// <summary>
+        /// Create an ILusidApiFactory using the specified secrets.json file
```

Comment:
~secrets.json~ - it can be any file name

Reply:
Should we create a kb article on secrets.json format ?